app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -71,16 +77,6 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
         searchTermTabGroups = requirePreference<SwitchPreference>(R.string.pref_key_search_term_tab_groups).also {
             it.isVisible = FeatureFlags.tabGroupFeature
             it.isChecked = it.context.settings().searchTermTabGroupsAreEnabled
-            it.onPreferenceChangeListener = SharedPreferenceUpdater()
-        }
-
-        radioManual = requirePreference(R.string.pref_key_close_tabs_manually)
-        radioOneMonth = requirePreference(R.string.pref_key_close_tabs_after_one_month)
-        radioOneWeek = requirePreference(R.string.pref_key_close_tabs_after_one_week)
-        radioOneDay = requirePreference(R.string.pref_key_close_tabs_after_one_day)
-
-        inactiveTabs = requirePreference<SwitchPreference>(R.string.pref_key_inactive_tabs).also {
-            it.isChecked = requireContext().settings().inactiveTabsAreEnabled
```

Comment:
Trying to understand why this was moved. Doesn't this mean the setOnPreferenceChangeListener will affect the searchTermTabGroups preference now?